modules/apps/foundation/file-uploads/file-uploads-configuration-test/src/testIntegration/java/com/liferay/file/uploads/configuration/verify/test/GeneralFileUploadsVerifyProcessTest.java

```diff
@@ -0,0 +1,185 @@
+
+		_bundleContext = bundle.getBundleContext();
+
+		ServiceReference<ConfigurationAdmin>
+			configurationAdminServiceReference =
+				_bundleContext.getServiceReference(ConfigurationAdmin.class);
+
+		_configurationAdmin = _bundleContext.getService(
+			configurationAdminServiceReference);
+	}
+
+	@AfterClass
+	public static void tearDownClass() throws Exception {
+		_serviceTracker.close();
+
+		Configuration configuration = _configurationAdmin.getConfiguration(
+			GeneralFileUploadsConfiguration.class.getName());
+
+		configuration.delete();
+	}
+
+	@Before
+	public void setUp() throws Exception {
+		super.setUp();
+
+		GeneralFileUploadsConfiguration baseGeneralFileUploadsConfiguration =
+			ConfigurableUtil.createConfigurable(
+				GeneralFileUploadsConfiguration.class, Collections.emptyMap());
+
+		Dictionary<String, Object> baseProperties = new Hashtable<>();
+
+		baseProperties.put(
+			PropsKeys.UPLOAD_SERVLET_REQUEST_IMPL_TEMP_DIR,
+			baseGeneralFileUploadsConfiguration.
+				uploadServletRequestImplTempDir());
+		baseProperties.put(
+			PropsKeys.UPLOAD_SERVLET_REQUEST_IMPL_MAX_SIZE,
+			baseGeneralFileUploadsConfiguration.
+				uploadServletRequestImplMaxSize());
+
+		Configuration baseConfiguration = _configurationAdmin.getConfiguration(
+			GeneralFileUploadsConfiguration.class.getName());
+
+		baseConfiguration.update(baseProperties);
+	}
+
+	@Test
+	public void testVerifyGeneralFileUploadsConfiguration() throws Exception {
+		GeneralFileUploadsConfiguration generalFileUploadsConfiguration =
+			getConfigurationModel();
+
+		final String testTempDir = StringUtil.randomString(10);
+		final long testUploadSize = RandomTestUtil.randomInt(1000, 5000);
+
+		Assert.assertNotEquals(
+			testTempDir,
+			generalFileUploadsConfiguration.uploadServletRequestImplTempDir());
+		Assert.assertNotEquals(
+			testUploadSize,
+			generalFileUploadsConfiguration.uploadServletRequestImplMaxSize());
+
+		PortletPreferences portletPreferences = PrefsPropsUtil.getPreferences();
+
+		portletPreferences.setValue(
+			PropsKeys.UPLOAD_SERVLET_REQUEST_IMPL_TEMP_DIR, testTempDir);
+		portletPreferences.setValue(
+			PropsKeys.UPLOAD_SERVLET_REQUEST_IMPL_MAX_SIZE,
+			String.valueOf(testUploadSize));
+
+		portletPreferences.store();
+
+		doVerify();
+
+		generalFileUploadsConfiguration = getConfigurationModel();
+
+		Assert.assertEquals(
+			testTempDir,
+			generalFileUploadsConfiguration.uploadServletRequestImplTempDir());
+		Assert.assertEquals(
+			testUploadSize,
+			generalFileUploadsConfiguration.uploadServletRequestImplMaxSize());
+	}
+
+	protected GeneralFileUploadsConfiguration getConfigurationModel()
+		throws Exception {
+
+		Configuration configuration = _configurationAdmin.getConfiguration(
+			GeneralFileUploadsConfiguration.class.getName());
+
+		Dictionary properties = configuration.getProperties();
+
+		return ConfigurableUtil.createConfigurable(
+			GeneralFileUploadsConfiguration.class, properties);
```

Comment:
Since we are just comparing the properties values here, we don't need a configuration object. We can just return the properties and get the value by `properties.get(configId)`.